bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2017.html

```diff
@@ -37,10 +37,10 @@ <h2 class="section-title" data-accordion-role="tab">{{ _('Government Demands for
           </tr>
           <tr>
             <th scope="row">
-              <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-subpoena' }}">{{ _('Subpoenas') }}</a>
+              <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-subpoena' }}">{{ _('Subpoenas*') }}</a>
```

Comment:
Instead of an asterisk we can do `<sup><a href="#footnote-1">1</a></sup>`. There's another one on this page, and we'll change that one to `footnote-2` so they appear in proper order.